frontend/src/components/modals/pointPopup/PointPopup.tsx

```diff
@@ -0,0 +1,291 @@
+import React, { useState } from 'react';
+import { styled } from '@mui/material/styles';
+import IconButton, { IconButtonProps } from '@mui/material/IconButton';
+import FavoriteIcon from '@mui/icons-material/Favorite';
+import ReportIcon from '@mui/icons-material/Report';
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import MoreVertIcon from '@mui/icons-material/MoreVert';
+import CloseIcon from '@mui/icons-material/Close';
+import IosShareIcon from '@mui/icons-material/IosShare';
+import ThumbUpIcon from '@mui/icons-material/ThumbUp';
+import ThumbDownIcon from '@mui/icons-material/ThumbDown';
+import CheckCircleTwoToneIcon from '@mui/icons-material/CheckCircleTwoTone';
+import SendIcon from '@mui/icons-material/Send';
+import {
+  Card,
+  CardHeader,
+  CardMedia,
+  CardContent,
+  CardActions,
+  Collapse,
+  Avatar,
+  Typography,
+  Box,
+  TextField,
+  FormControl,
+  Button,
+  List,
+  ListItem,
+  Divider,
+  ListItemText,
+  ListItemAvatar
+} from '@mui/material';
+
+import './pointPopup.scss';
```

Comment:
we don't use scss

Reply:
I didn't know we don't use scss. I'll fix it. But it seems to me that this gives additional opportunities for more flexible customization.

Reply:
> I didn't know we don't use scss. I'll fix it. But it seems to me that this gives additional opportunities for more flexible customization.

Maybe) But it's violates code convention too.